pack/tdc/tdcp.json

```
@@ -0,0 +1,515 @@
+        "traits": "Tactic. Improvised.",
+        "type_code": "event",
+        "xp": 2
+    },
+    {
+        "pack_code": "tdcp",
+        "code": "11099",
+        "position": 99,
+        "cost": 2,
+        "deck_limit": 1,
+        "faction_code": "neutral",
+        "illustrator": "Pixoloid Studios",
+        "is_unique": true,
+        "name": "Library Pass",
+        "quantity": 1,
+        "restrictions": "trait:miskatonic, trait:scholar",
+        "text": "[[Miskatonic]], [[Scholar]] deck only. Limit 1 per deck.\n[fast] During your turn, if there is no asset attached to Library Pass, spend 1 resource: Attach a [[Tome]] asset from your hand to this card. Attached asset takes up no slots. Limit 1 attached asset.\n<b>Forced</b> - When your turn ends: Place attached asset on the bottom of your deck.",
+        "traits": "Item. Miskatonic.",
+        "type_code": "asset",
+        "xp": 1
+    },
+    {
+        "code": "11106",
+        "cost": 0,
+        "deck_limit": 2,
+        "faction_code": "neutral",
+        "illustrator": "Lin Hsiang",
+        "name": "True Awakening",
+        "pack_code": "tdcp",
+        "position": 106,
+        "quantity": 2,
+        "restrictions": "trait:blessed, trait:chosen, trait:drifter",
+        "skill_wild": 1,
+        "text": "[[Blessed]], [[Chosen]], [[Drifter]] deck only.\nChoose and resolve two of the following, in any order:\n- Draw 2 cards.\n- Discover 1 clue at your location.\n- Heal 1 damage.\n- Heal 1 horror.",
+        "traits": "Insight. Spirit.",
+        "type_code": "event",
+        "xp": 2
+    },
+    {
+        "code": "11111",
+        "cost": 2,
+        "deck_limit": 2,
+        "faction_code": "neutral",
+        "illustrator": "Lin Hsiang",
+        "name": "Nose to the Grindstone",
+        "pack_code": "tdcp",
+        "position": 111,
+        "quantity": 2,
+        "restrictions": "trait:criminal, trait:drifter, trait:entrepreneur",
+        "skill_intellect": 2,
+        "text": "[[Criminal]], [[Drifter]], [[Entrepreneur]] deck only.\n[fast] During a skill test on a [[Tool]] asset, exhaust Nose to the Grindstone: You get +2 skill value for this skill test. If this skill test fails, either replenish 1 supply on that asset or gain 1 resource.",
+        "traits": "Talent.",
+        "type_code": "asset",
+        "xp": 3
+    },
+    {
+        "code": "11117",
+        "cost": 4,
+        "deck_limit": 2,
+        "faction_code": "neutral",
+        "flavor": "\"'Publish or Perish,' they said. Perhaps we'll do both.\"",
+        "health": 2,
+        "illustrator": "Alexandre Dainche",
+        "is_unique": true,
+        "name": "Archibald MacVeigh",
+        "pack_code": "tdcp",
+        "position": 117,
+        "quantity": 2,
+        "restrictions": "trait:assistant, trait:reporter, trait:scholar",
+        "sanity": 3,
+        "skill_intellect": 1,
+        "skill_wild": 1,
+        "slot": "Ally",
+        "subname": "Beleagured Lecturer",
```

```suggestion
        "subname": "Beleaguered Lecturer",
```

I'm pretty sure it's a typo so I'm putting in the correction here.